src/components/carousel/index.tsx

```diff
@@ -68,7 +68,7 @@ const Carousel: FC<Props> = (props) => {
   const [mainCarouselRef, mainCarousel] = useEmblaCarousel({
     loop: true,
     startIndex: startIndex,
-    speed: 20,
+    duration: 20,
```

Comment:
`speed` has been removed from the API as it depended on frame rate.

The new `duration` option docs can be found here: https://www.embla-carousel.com/api/options/#duration

I've set the value to `20` is it is the minimum recommended value. This makes the transition last longer than previously, however, the animation seems smooth and fast enough. 

I've also tested with smaller values, but they make the transition go past the target destination and then bounce back a couple of times (since the library has a new primitive physics engine to handle this).

Reply:
can you check this with a designer? For me it seems to be way too slow. Maybe it is because I am used to the old behavior though